include/cuco/detail/open_addressing_impl.cuh

```diff
@@ -135,6 +137,55 @@ class open_addressing_impl {
     this->clear_async(stream);
   }
 
+  /**
+   * @brief Constructs a statically-sized open addressing data structure with the number of elements
+   * to insert, sentinel values, the desired load factor and CUDA stream
+   *
+   * @note The actual capacity depends on the given `n`, the probing scheme, CG size, the desired
+   * load factor and the window size and it is computed via the `make_window_extent` factory. Insert
+   * operations will not automatically grow the container. Attempting to insert more unique keys
+   * than the capacity of the container results in undefined behavior.
+   * @note Any `*_sentinel`s are reserved and behavior is undefined when attempting to insert
+   * this sentinel value.
+   * @note If a non-default CUDA stream is provided, the caller is responsible for synchronizing the
```

Comment:
Why do I need to synchronize if I provide a non default stream? Why can't I construct on a stream and insert on that same stream without syncing?

Reply:
I wanted to provide more specific instructions for the non-default stream use case but looks like it doesn't help. Rephrase it as `This constructor doesn't synchronize the given stream`